identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/screen/forgetPassword/ForgetPasswordScreenInteractionListener.kt

```diff
@@ -0,0 +1,15 @@
+package net.thechance.mena.identity.presentation.screen.forgetPassword
+
+import net.thechance.mena.identity.presentation.base.BaseInteractionListener
+import net.thechance.mena.identity.presentation.bottomSheet.countryPicker.menaCountries.MenaCountry
+
+interface ForgetPasswordScreenInteractionListener : BaseInteractionListener{
+    fun onSelectCountryItem(country: MenaCountry)
+    fun onClickConfirmButton()
+    fun onDismissBottomSheet()
+    fun onContinueClicked()
+    fun onPhoneCodeClicked()
+    fun onPhoneChanged(phone: String)
+    fun onBackClicked()
```

Comment:
please rename it to onClick not Clicked
```suggestion
    fun onClickPhoneCode()
    fun onChangePhone(phone: String)
    fun onClickBack()
```